### PR TITLE
fix openwindow & opentab

### PR DESCRIPTION
**Pull request by paulrouget**
Dont' review yet.

### CODE REVIEW

**Comment by paulrouget** [comment]
Not sure this is working well.

Here are the 3 things to test:
- Cmd-Click
- window.open
- window open from a different application

File: src/browser/web-view.js [hunk 1]
```diff
@@ -278,13 +280,9 @@ define((require, exports, module) => {
    });
  };
 
-  const update = (state, action) =>
-    action instanceof Load ?
+  const update = (state, action) => {
+    return action instanceof Load ?
```
Comment (gordonbrander) [inline src/browser/web-view.js]: Do we need to wrap this in `{}` ?

File: src/browser/web-view.js [hunk 2]
```diff
@@ -439,7 +438,7 @@ define((require, exports, module) => {
           features: detail.features});
 
   Event.mozbrowseropentab = ({detail}) =>
-    OpenInBackground({uri: detail.uri});
+    OpenInBackground({uri: detail.url});
```
Comment (gordonbrander) [inline src/browser/web-view.js]: `url` -> `uri`. Is this correct?

**Comment by gordonbrander** [comment]
@Gozala provided a fix https://github.com/mozilla/browser.html/pull/492

**Pull request closed without merging**